kasa/tests/smart/modules/test_contact.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+
+from kasa.smart.modules import ContactSensor
+from kasa.tests.device_fixtures import parametrize
+
+contact = parametrize(
+    "has humidity", model_filter="T110", protocol_filter={"SMART.CHILD"}
+)
+
+
+@contact
+@pytest.mark.parametrize(
+    "feature, type",
+    [
+        ("is_open", bool),
+    ],
+)
```

Comment:
It's using the same template as other feature tests, so that's why I thought we should leave it as is for the time being. This should probably be generalized to avoid the copy&paste code at some point.